src/content/configuration/configuration-languages.md

```diff
@@ -39,6 +40,46 @@ const config: webpack.Configuration = {
 export default config;
 ```
 
+next check your tsconfig.json:
+
+If module in compilerOptions in tsconfig.json is commonjs, setting is complete, else webpack getting error.
```

Comment:
@youta1119 This is missing quite a few [articles](https://en.wikipedia.org/wiki/English_articles), for example…

> If **the** module in compilerOptions in tsconfig.json is commonjs, **the** setting is complete, else webpack will get an error.

Have a look through the rest of your PR and happy to help if you get stuck.